data/language/en-GB.txt

```diff
@@ -2448,7 +2448,7 @@ STR_3357    :{WINDOW_COLOUR_2}Are you sure you want to permanently delete {STRIN
 STR_3358    :Can't delete track design...
 STR_3359    :{BLACK}No track designs of this type
 STR_3360    :Warning!
-STR_3361    :Too many track designs of this type - Some will not be listed.
+STR_3361    :Too many track designs of this type - some will not be listed
```

Comment:
Same as above, keep capitalization, remove period.  This one is also basically two sentences, so you could change it to a period instead of a -.